[2020-12-27T21:42:09-0800] [MainThread] [I] [toil] Running Toil version 4.2.0-3aa1da130141039cb357efe36d7df9b9f6ae9b5b on host mustard.
[2020-12-27T21:42:09-0800] [MainThread] [I] [toil.leader] Issued job 'parse_config_file' kind-parse_config_file/instance-fyss1ild with job batch system ID: 0 and cores: 1, disk: 2.0 G, and memory: 2.0 G
[2020-12-27T21:42:10-0800] [MainThread] [I] [toil.worker] Redirecting logging to /scratch/drkthomp/workDir/node-addd4423-93ae-496c-9a57-c39eb34ed766-e8611b53-bdab-4598-acae-87977f99831e/tmposj3xp52/worker_log.txt
[2020-12-27T21:42:11-0800] [Thread-1  ] [E] [toil.batchSystems.singleMachine] Got exit code 1 (indicating failure) from job _toil_worker parse_config_file file:/scratch/drkthomp/jobStore kind-parse_config_file/instance-fyss1ild.
[2020-12-27T21:42:11-0800] [MainThread] [W] [toil.leader] Job failed with exit value 1: 'parse_config_file' kind-parse_config_file/instance-fyss1ild
[2020-12-27T21:42:11-0800] [MainThread] [W] [toil.leader] The job seems to have left a log file, indicating failure: 'parse_config_file' kind-parse_config_file/instance-fyss1ild
[2020-12-27T21:42:11-0800] [MainThread] [W] [toil.leader] Log from job kind-parse_config_file/instance-fyss1ild follows:
=========>
	[2020-12-27T21:42:10-0800] [MainThread] [I] [toil.worker] ---TOIL WORKER OUTPUT LOG---
	[2020-12-27T21:42:10-0800] [MainThread] [I] [toil] Running Toil version 4.2.0-3aa1da130141039cb357efe36d7df9b9f6ae9b5b on host mustard.
	[2020-12-27T21:42:10-0800] [MainThread] [I] [toil.fileStores.abstractFileStore] LOG-TO-MASTER: Parsing config file
	/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py:333: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
	  input_config = yaml.load(conf.read())
	/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py:338: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
	  required_keys = yaml.load(ref.read())
	/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py:350: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
	  protect_defaults = yaml.load(pdf.read())
	Traceback (most recent call last):
	  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/worker.py", line 368, in workerScript
	    job._runner(jobGraph=jobGraph, jobStore=jobStore, fileStore=fileStore, defer=defer)
	  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/job.py", line 1424, in _runner
	    returnValues = self._run(jobGraph, fileStore)
	  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/job.py", line 1361, in _run
	    return self.run(fileStore)
	  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/job.py", line 1565, in run
	    rValue = userFunction(*((self,) + tuple(self._args)), **self._kwargs)
	  File "/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py", line 444, in parse_config_file
	    max_cores)
	  File "/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py", line 367, in _parse_config_file
	    not input_config['mutation_calling']['star_fusion']['run'])
	  File "/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py", line 304, in parse_patients
	    output_dict[f + '_fastq_1'])
	  File "/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py", line 236, in get_fastq_2
	    'of the form <fastq_prefix>1.[fq/fastq][.gz]?' % fastq_1)
	protect.common.ParameterError: Could not determine prefix from provided fastq (/scratch/drkthomp/fullbams/gerald_C1TD1ACXX_8_ACAGTG_R2.fastq). Is it of the form <fastq_prefix>1.[fq/fastq][.gz]?
	[2020-12-27T21:42:10-0800] [MainThread] [E] [toil.worker] Exiting the worker because of a failed job on host mustard
<=========
[2020-12-27T21:42:11-0800] [MainThread] [W] [toil.jobGraph] Due to failure we are reducing the remaining retry count of job 'parse_config_file' kind-parse_config_file/instance-fyss1ild with ID kind-parse_config_file/instance-fyss1ild to 1
[2020-12-27T21:42:11-0800] [MainThread] [I] [toil.leader] 0 jobs are running, 0 jobs are issued and waiting to run
[2020-12-27T21:42:11-0800] [MainThread] [I] [toil.leader] Issued job 'parse_config_file' kind-parse_config_file/instance-fyss1ild with job batch system ID: 1 and cores: 1, disk: 2.0 G, and memory: 2.0 G
[2020-12-27T21:42:11-0800] [MainThread] [I] [toil.worker] Redirecting logging to /scratch/drkthomp/workDir/node-addd4423-93ae-496c-9a57-c39eb34ed766-e8611b53-bdab-4598-acae-87977f99831e/tmpjgxfvtqr/worker_log.txt
[2020-12-27T21:42:12-0800] [Thread-1  ] [E] [toil.batchSystems.singleMachine] Got exit code 1 (indicating failure) from job _toil_worker parse_config_file file:/scratch/drkthomp/jobStore kind-parse_config_file/instance-fyss1ild.
[2020-12-27T21:42:12-0800] [MainThread] [W] [toil.leader] Job failed with exit value 1: 'parse_config_file' kind-parse_config_file/instance-fyss1ild
[2020-12-27T21:42:12-0800] [MainThread] [W] [toil.leader] The job seems to have left a log file, indicating failure: 'parse_config_file' kind-parse_config_file/instance-fyss1ild
[2020-12-27T21:42:12-0800] [MainThread] [W] [toil.leader] Log from job kind-parse_config_file/instance-fyss1ild follows:
=========>
	[2020-12-27T21:42:11-0800] [MainThread] [I] [toil.worker] ---TOIL WORKER OUTPUT LOG---
	[2020-12-27T21:42:11-0800] [MainThread] [I] [toil] Running Toil version 4.2.0-3aa1da130141039cb357efe36d7df9b9f6ae9b5b on host mustard.
	[2020-12-27T21:42:12-0800] [MainThread] [I] [toil.fileStores.abstractFileStore] LOG-TO-MASTER: Parsing config file
	/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py:333: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
	  input_config = yaml.load(conf.read())
	/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py:338: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
	  required_keys = yaml.load(ref.read())
	/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py:350: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
	  protect_defaults = yaml.load(pdf.read())
	Traceback (most recent call last):
	  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/worker.py", line 368, in workerScript
	    job._runner(jobGraph=jobGraph, jobStore=jobStore, fileStore=fileStore, defer=defer)
	  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/job.py", line 1424, in _runner
	    returnValues = self._run(jobGraph, fileStore)
	  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/job.py", line 1361, in _run
	    return self.run(fileStore)
	  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/job.py", line 1565, in run
	    rValue = userFunction(*((self,) + tuple(self._args)), **self._kwargs)
	  File "/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py", line 444, in parse_config_file
	    max_cores)
	  File "/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py", line 367, in _parse_config_file
	    not input_config['mutation_calling']['star_fusion']['run'])
	  File "/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py", line 304, in parse_patients
	    output_dict[f + '_fastq_1'])
	  File "/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py", line 236, in get_fastq_2
	    'of the form <fastq_prefix>1.[fq/fastq][.gz]?' % fastq_1)
	protect.common.ParameterError: Could not determine prefix from provided fastq (/scratch/drkthomp/fullbams/gerald_C1TD1ACXX_8_ACAGTG_R2.fastq). Is it of the form <fastq_prefix>1.[fq/fastq][.gz]?
	[2020-12-27T21:42:12-0800] [MainThread] [E] [toil.worker] Exiting the worker because of a failed job on host mustard
<=========
[2020-12-27T21:42:12-0800] [MainThread] [W] [toil.jobGraph] Due to failure we are reducing the remaining retry count of job 'parse_config_file' kind-parse_config_file/instance-fyss1ild with ID kind-parse_config_file/instance-fyss1ild to 0
[2020-12-27T21:42:12-0800] [MainThread] [W] [toil.leader] Job 'parse_config_file' kind-parse_config_file/instance-fyss1ild with ID kind-parse_config_file/instance-fyss1ild is completely failed
[2020-12-27T21:42:15-0800] [MainThread] [I] [toil.leader] Finished toil run with 1 failed jobs.
[2020-12-27T21:42:15-0800] [MainThread] [I] [toil.leader] Failed jobs at end of the run: 'parse_config_file' kind-parse_config_file/instance-fyss1ild
Traceback (most recent call last):
  File "/private/home/drkthomp/protect/venv/bin/ProTECT", line 11, in <module>
    load_entry_point('protect', 'console_scripts', 'ProTECT')()
  File "/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py", line 1028, in main
    Job.Runner.startToil(start, params)
  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/job.py", line 796, in startToil
    return toil.start(job)
  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/common.py", line 846, in start
    return self._runMainLoop(rootJobGraph)
  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/common.py", line 1138, in _runMainLoop
    jobCache=self._jobCache).run()
  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/leader.py", line 269, in run
    raise FailedJobsException(self.config.jobStore, self.toilState.totalFailedJobs, self.jobStore)
toil.leader.FailedJobsException: The job store 'file:/scratch/drkthomp/jobStore' contains 1 failed jobs: 'parse_config_file' kind-parse_config_file/instance-fyss1ild
Log from job 'parse_config_file' kind-parse_config_file/instance-fyss1ild follows:
=========>
	[2020-12-27T21:42:11-0800] [MainThread] [I] [toil.worker] ---TOIL WORKER OUTPUT LOG---
	[2020-12-27T21:42:11-0800] [MainThread] [I] [toil] Running Toil version 4.2.0-3aa1da130141039cb357efe36d7df9b9f6ae9b5b on host mustard.
	[2020-12-27T21:42:12-0800] [MainThread] [I] [toil.fileStores.abstractFileStore] LOG-TO-MASTER: Parsing config file
	/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py:333: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
	  input_config = yaml.load(conf.read())
	/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py:338: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
	  required_keys = yaml.load(ref.read())
	/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py:350: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
	  protect_defaults = yaml.load(pdf.read())
	Traceback (most recent call last):
	  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/worker.py", line 368, in workerScript
	    job._runner(jobGraph=jobGraph, jobStore=jobStore, fileStore=fileStore, defer=defer)
	  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/job.py", line 1424, in _runner
	    returnValues = self._run(jobGraph, fileStore)
	  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/job.py", line 1361, in _run
	    return self.run(fileStore)
	  File "/private/home/drkthomp/protect/venv/lib/python3.6/site-packages/toil/job.py", line 1565, in run
	    rValue = userFunction(*((self,) + tuple(self._args)), **self._kwargs)
	  File "/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py", line 444, in parse_config_file
	    max_cores)
	  File "/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py", line 367, in _parse_config_file
	    not input_config['mutation_calling']['star_fusion']['run'])
	  File "/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py", line 304, in parse_patients
	    output_dict[f + '_fastq_1'])
	  File "/private/home/drkthomp/protect/src/protect/pipeline/ProTECT.py", line 236, in get_fastq_2
	    'of the form <fastq_prefix>1.[fq/fastq][.gz]?' % fastq_1)
	protect.common.ParameterError: Could not determine prefix from provided fastq (/scratch/drkthomp/fullbams/gerald_C1TD1ACXX_8_ACAGTG_R2.fastq). Is it of the form <fastq_prefix>1.[fq/fastq][.gz]?
	[2020-12-27T21:42:12-0800] [MainThread] [E] [toil.worker] Exiting the worker because of a failed job on host mustard
<=========
